doc_admin/AdminManual.adoc

```diff
@@ -1024,6 +1024,58 @@ By default, DEBUG is set to no.
 _*[.underline]#NB#*: An application restart is required in order to see the modification applied._
 
 
+==== 5.1.31 ALLOWMULTIPLEOPENEDBILL
+
+The following table shows the default value and the allowed ones:
+
+[cols=",,",options="header",]
+|===
+|key |default value |allowed values
+|DEBUG |no |yes, no
```

Comment:
Fix with correct key value ALLOWMULTIPLEOPENEDBILL

Reply:
Ok I see the error